scss/settings/_settings.scss

```diff
@@ -73,6 +73,7 @@ $global-radius: 0;
 $global-text-direction: ltr;
 $global-flexbox: false;
 $print-transparent-backgrounds: true;
+$contrast-warnings: true;
```

Comment:
@IamManchanda I just pushed up the requested change. The variable is no longer being included in the _settings.scss file.